recipes/jupyterlab_launcher/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2015-2016, Project Jupyter Contributors
```

Comment:
This can be dropped as we can pull it directly from the source. No need to maintain it here in addition.